### ISSUE TITLE
Make missing icons for new cup types in CustomMedals

### ISSUE THREAD

**Issue by minj** [minj/foxtrick#1290]
We need at least 4 icons for CupB1, CupB2, CupB3, CupC. CupA could be re-purposed from the old cup icon, I guess.

---
**From:** kopsterkespits
**PostID:** [16412248.359](https://www.hattrick.org/goto.ashx?path=%2FForum%2FRead.aspx%3Ft%3D16412248%26n%3D359%26v%3D0)
**To:** Everyone
**Re:** [16412248.358](https://www.hattrick.org/goto.ashx?path=%2FForum%2FRead.aspx%3Ft%3D16412248%26n%3D358%26v%3D0)
**Datetime:** 2015-03-22 11:20
**Message:**

I have old medals thicked because I like them way better than the ugly 'new' ones. But... after 10 years of struggeling I finally won a price but it's not displayed in my price shelve. Is there a way to fix this or can someone make 'new old medals' please?

In case someone does I think the secondary cups should be smaller.

---
**From:** kopsterkespits
**PostID:** [16412248.365](https://www.hattrick.org/goto.ashx?path=%2FForum%2FRead.aspx%3Ft%3D16412248%26n%3D365%26v%3D0)
**To:** LA-MJ
**Re:** [16412248.364](https://www.hattrick.org/goto.ashx?path=%2FForum%2FRead.aspx%3Ft%3D16412248%26n%3D364%26v%3D0)
**Datetime:** 2015-03-28 11:43
**Message:**

[(211465)](https://www.hattrick.org/goto.ashx?path=%2FClub%2F%3FTeamID%3D211465) this team has the normal cupmedal.
Mine has the new one, lol blank atm...

I did try to 'green up' a secondary cup and make it a bit smaller.

[(http://i57.tinypic.com/2h4hs0g.gif)](http://i57.tinypic.com/2h4hs0g.gif)

[![Įkeltas paveikslėlis](http://i57.tinypic.com/2h4hs0g.gif)](http://i57.tinypic.com/2h4hs0g.gif)

is this useful?

I don't get it right now if they can be replaced or not btw.

---
> FoxTrick 0.14.0.3 release mozilla - Stage: false - Skin: normal - nl - ltr - 1920 x 1040 - Mozilla/5.0 (Windows NT 6.1; WOW64; rv:36.0) Gecko/20100101 Firefox/36.0 - Win32

**Comment by minj** [comment]
For testing:
**NEW:** [(211465)](https://www.hattrick.org/goto.ashx?path=%2FClub%2F%3FTeamID%3D211465) 
**OLD:** [(145644)](https://www.hattrick.org/goto.ashx?path=/Club/?TeamID=145644)

**Comment by ratsrats** [comment]
"We need at least 4 icons for CupB1, CupB2, CupB3, CupC."

Here are some proposals:
![cupb1_029](https://cloud.githubusercontent.com/assets/5188518/7190257/11794878-e485-11e4-846e-4695b5db4f96.png)
![cupb2_028](https://cloud.githubusercontent.com/assets/5188518/7190262/1aa86cb2-e485-11e4-97d5-cdbf598d5aaf.png)
![cupb3_027](https://cloud.githubusercontent.com/assets/5188518/7190268/22270cd2-e485-11e4-92f4-3b143b9918a4.png)
![cupc_026](https://cloud.githubusercontent.com/assets/5188518/7190273/277db12c-e485-11e4-9a34-fef8e748e52b.png)

**Comment by minj** [comment]
@ratsrats I think you missed the point... They should look like old medals, not new :)

**Comment by ratsrats** [comment]
Forum »   FoxTrick »   Developer 
Murkos [post=16281487.387] displayed as first cup:

Old-type cup in season 32:
(/App_Themes/Standard/images/trophies/leaguecup_...)

I took that one as a starting point. "Old-type cup" could hardly be interpreted as "new".
Could someone please get the confusion out of the way and explain - with an example - where to start from.

**Comment by minj** [comment]
The ones you see in that message are made by HTs and you can see them with or without Foxtrick. They are 'new-style' trophies.

Foxtrick module CustomMedals brings back the 'old-style' trophies. E.g. trophy list for [(211465)](https://www.hattrick.org/goto.ashx?path=%2FClub%2F%3FTeamID%3D211465) looks like this:
![](http://i.imgur.com/PkyvNRe.png)

Now, 'new' cups(=tournament type) refer to the new cup system. Cup was changed into CupA, and cups B1,B2,B3,C were added. Each of them got a new-style trophy icon.

The problem is we don't have 'old-style' trophies for new cups.

**Comment by minj** [comment]
You can the 'old-style' trophies we already have here: 
https://github.com/minj/foxtrick/tree/master/content/resources/img/custommedals/oldhtmedals

**Comment by ratsrats** [comment]
So the old league cup = cupA which remains as is, equals the first cup from your example.
cups B1-2-3 and C are to be designed, to accomdate user requests, in style of the old league cup, cupA now.

**Comment by minj** [comment]
precisely

2015-04-17 16:20 GMT+03:00, ratsrats <notifications@github.com>:

**Comment by ratsrats** [comment]
Here is a cupB1 proposal
![cupb1_15](https://cloud.githubusercontent.com/assets/5188518/7207703/ae4b6846-e53b-11e4-96b8-cacbfd9b2968.png)

**Comment by ratsrats** [comment]
The width - 34 px - is adapted to the width of the new ones, as is the color.

B2 will be with one ear, in red,
B3 will be with no ears, in blue,
C1 will have no ears and no lid.

**Comment by minj** [comment]
Looks good I think. I don't know about the size though. It seems to me
they should not be larger than cupA (ex cup). Unless you would like to
redo that one as well ;)

2015-04-17 21:01 GMT+03:00, ratsrats <notifications@github.com>:

**Comment by ratsrats** [comment]
You are right about the size. The cupA sould be 34 px wide as well.

The old widths are extreme small; @ 34 px the cups have a bit more 'breathing' room. That way they will fit better in the website layout.

**Comment by ratsrats** [comment]
Here follows a missing cup proposal
![cupa_10_01](https://cloud.githubusercontent.com/assets/5188518/7229222/7f51934e-e761-11e4-9862-ef66b98e7c72.png)
![cupb1_10_02](https://cloud.githubusercontent.com/assets/5188518/7229229/8ffe7a54-e761-11e4-8ac1-0ae01d251190.png)
![cupb2_10_03](https://cloud.githubusercontent.com/assets/5188518/7229235/a37e54aa-e761-11e4-8c85-ce7bea03950b.png)
![cupb3_10_03](https://cloud.githubusercontent.com/assets/5188518/7229238/acaad77e-e761-11e4-8a5b-2fe7cd9e8446.png)
![cupc_10_05](https://cloud.githubusercontent.com/assets/5188518/7229240/b0e78936-e761-11e4-8444-4546c2c5b7d7.png)

**Comment by ratsrats** [comment]
From top to bottom:
cupA, cupB1, cupB2, cupB3, cupC
Width is 34 px, colors, chosen from the new cups.

It could be a no-issue:
I do not have copyrights to these images. If you think it would be necessary, you should give it more attention.

![trophyleaguecup1](https://cloud.githubusercontent.com/assets/5188518/7229330/cc4efbea-e762-11e4-81cf-3010b4655671.png)
You could still be choosing the old cup @ 34px ; )

**Comment by ratsrats** [comment]
![cupa_10_02](https://cloud.githubusercontent.com/assets/5188518/7229353/fdd2b7ec-e762-11e4-9d8c-937a6e35731a.png)

**Comment by minj** [comment]
Looks like you have a universal seal of approval :)

**Comment by jetwitaussi** [comment]
great!

**Comment by ratsrats** [comment]
Please replace cupB2 with following, a couple of px got lost in the last one.
![cupb2_10_03](https://cloud.githubusercontent.com/assets/5188518/7248647/9cee86ec-e814-11e4-8519-81e071402b89.png)

**Comment by ratsrats** [comment]
It has been my pleasure!

**Comment by minj** [comment]
The problem with that module is that it is hard-coded for 30x30 gifs. While using gifs is not that bad in this case I guess (quite a size benefit in truth) I would need to rewrite it slightly for custom image sizes.

Anyway here's how it would look:
![](http://i.imgur.com/P86FGyF.png)

Personally I don't have an opinion either way but some could find the size difference disturbing. Down-scaling to 30px would also make my life easier ;)

**Comment by ratsrats** [comment]
No problem, if you prefer the 30 px gifs, I'll produce those. 

The cup is a sweet design. I never liked the shields though, they look more like weard ducks ; ) , hardly to distinguish one from another.

**Comment by ratsrats** [comment]
An other option, to do things right, would be to enlarge the shields as well to 34 px wide. That way all medals would fit better into the HT page lay-out. 

@minj 
If you decide to code for 34 px wide, I will supply you with the resized shields.

**Comment by minj** [comment]
@ratsrats sorry for taking the time to respond. I was working on Links modules. I also tried to elicit some feedback from HT forums but few people care it seems :)

Somebody commented that the cups look a bit pixelated and I agree. I think it's best to do them as 30x30. Do you have hires versions that you could scale down or should I just do it myself?

**Comment by minj** [comment]
Here's how it looks: 
![](https://cloud.githubusercontent.com/assets/1035523/7535044/84526414-f589-11e4-98a6-7a4ae8f48dd0.png)

**Comment by ratsrats** [comment]
When you are happy, we all are happy ; )

The pixelation comes from the enlarging from 30 --> 34 px. There is no hires, it all starts with the 30 px wide icons.

**Comment by minj** [comment]
@ratsrats if you still feel like helping out, please take a look at #1286 next.
There are quite a few match events that could benefit from additional icons.

**Comment by minj** [comment]
It seems I was a bit optimistic with this. There are a bunch more different trophies HTs use.

trophyCupDivisional7A:
![](https://stage.hattrick.org/App_Themes/Standard/images/trophies/cupDiv7.png)
trophyCupDivisional8A:
![](https://stage.hattrick.org/App_Themes/Standard/images/trophies/cupDiv8.png)
trophyCupDivisional9A:
![](https://stage.hattrick.org/App_Themes/Standard/images/trophies/cupDiv9.png)
trophyCupDivisionalB1:
![](https://stage.hattrick.org/App_Themes/Standard/images/trophies/cupDivB1.png)
trophyCupDivisionalB2:
![](https://stage.hattrick.org/App_Themes/Standard/images/trophies/cupDivB2.png)
trophyCupDivisionalB3:
![](https://stage.hattrick.org/App_Themes/Standard/images/trophies/cupDivB3.png)

trophySupporterWeekTrophyWinner:
![](https://stage.hattrick.org/App_Themes/Standard/images/trophies/supporterWeekTrophy.png)
trophyTournamentWinner1:
![](https://stage.hattrick.org/App_Themes/Standard/images/trophies/tournament_1.png)
trophyTournamentWinner2:
![](https://stage.hattrick.org/App_Themes/Standard/images/trophies/tournament_2.png)
trophyTournamentWinner3:
![](https://stage.hattrick.org/App_Themes/Standard/images/trophies/tournament_3.png)

tutorialResult1:
![](https://stage.hattrick.org/App_Themes/Standard/images/trophies/tutorial_1.png)
tutorialResult2:
![](https://stage.hattrick.org/App_Themes/Standard/images/trophies/tutorial_2.png)
tutorialResult3:
![](https://stage.hattrick.org/App_Themes/Standard/images/trophies/tutorial_3.png)
tutorialResult4:
![](https://stage.hattrick.org/App_Themes/Standard/images/trophies/tutorial_4.png)

**Comment by ratsrats** [comment]
So, 11 new cups old style are requested and 3 tutorial results.
I'll think of something appropriate ; )

**Comment by minj** [comment]
I don't actually know where, how and why tutorialResult* are used, if we can actually see them and if they really need to be made. I just found them on HT list...

**Comment by ratsrats** [comment]
where can I find the CupA, B1, B2, B3 C cups as modified by yourself?

**Comment by minj** [comment]
same as previously: https://github.com/minj/foxtrick/tree/master/content/resources/img/custommedals/oldhtmedals

**Comment by ratsrats** [comment]
Here follows a missing cup proposal:
![trophycupnational_7a](https://cloud.githubusercontent.com/assets/5188518/9199417/8f0b948a-4042-11e5-9198-2bfe28d14bf0.gif)
![trophycupnational_8a](https://cloud.githubusercontent.com/assets/5188518/9199418/8f0ebfe8-4042-11e5-83fb-a8af7aa20722.gif)
![trophycupnational_9a](https://cloud.githubusercontent.com/assets/5188518/9199419/8f131d7c-4042-11e5-8655-5548aa0177cd.gif)
![trophycupnational_b1](https://cloud.githubusercontent.com/assets/5188518/9199420/8f1a059c-4042-11e5-823b-ed14c477602c.gif)
![trophycupnational_b2](https://cloud.githubusercontent.com/assets/5188518/9199421/8f22e824-4042-11e5-9b3e-331c6702a817.gif)
![trophycupnational_b3](https://cloud.githubusercontent.com/assets/5188518/9199422/8f2a49b6-4042-11e5-91bc-619c6dac75e0.gif)

**Comment by minj** [comment]
@ratsrats sorry for not getting back to you for so long. I wanted to get some opinions first and since so few people visit github I always cross-post stuff like that in HT forums. Even though I tried to encourage discussion on HT forums a couple of time there were few taker, however.

The show must go on, though, so I channel my time where I am needed most/am best at. As you may be aware I had to make new infrastructure to adhere to Mozilla's addon signing policy. Additionally HTs constantly make new changes that break things all the time but everybody and their gradma just want everything working smoothly as well as their new ideas implemented ASAP.

TL;DR maintaining Foxtrick alone is hard.

----

Now as for your cup proposals, people did not particularly like the numbering system. I am not specifically against it but I think a more pleasingly-looking font would be nice.

However, the B cups don't look right. I think the whole system needs a bit of tweak. I have some ideas if you can spare more of your time and talent :)

**Comment by minj** [comment]
TODO:
- [ ] main B* cups need to look the same, different colors only. This should probably be intermediate type (=currently red).
- [ ] lower league B cups need to also look the same, different colors only. This should probably be low type (=currently blue).
- [ ] lower league A cups need to look the same as B cups above, only purple + roman numeral
- [ ] trophy cups need new designs

**Comment by ratsrats** [comment]
Hi, 
I'm completely caught by surprise ; ) missed your 6 oct post. Forgot all about it.
So what is it all about? About cups ; )

When you could provide me with superfluous detailed information in pictures, I'll be happy to be of assistance. 
.

**Comment by minj** [comment]
Welcome back @ratsrats :+1:

Well the TODO list pretty much covers it:
- `trophyCupB*` cups are of the same rank and need to look the same, only different colors (red, green, blue). I think they should all have one 'ear' like the red one above so blue & green should be remade.
- `trophyCupDivisionalB*` cups should also be of the same (but lower rank) and I think they should have no ears but with one pom pom like the blue one above so green & red should be done accordingly.
- `trophyCupDivisional*A` should look like `trophyCupDivisionalB*` but purple and with roman numerals like HT originals up above.
- need some new ideas for `trophySupporterWeekTrophyWinner` and `trophyTournamentWinner*`

**Comment by ratsrats** [comment]
I haven't forgotten.

Please be patient  : )

**Comment by ratsrats** [comment]
Some ideas are waiting to be developed, please hang on.

**Comment by ratsrats** [comment]
![medaille 01 gold](https://cloud.githubusercontent.com/assets/5188518/11748637/77ce73d6-a029-11e5-906a-c8e37118c4bc.jpg)
![cup 1c](https://cloud.githubusercontent.com/assets/5188518/11748638/77d67c34-a029-11e5-915b-9830b0cd07e3.jpg)
![cup 1b](https://cloud.githubusercontent.com/assets/5188518/11748639/77de5170-a029-11e5-90b6-0543be5b6bed.jpg)
![cup 1a](https://cloud.githubusercontent.com/assets/5188518/11748640/77e3e7de-a029-11e5-8de3-96dd44a752f9.jpg)
![cup 04](https://cloud.githubusercontent.com/assets/5188518/11748641/77ed7fce-a029-11e5-8520-3809f3bd2951.jpg)
![cup 03](https://cloud.githubusercontent.com/assets/5188518/11748642/77f36830-a029-11e5-97c6-cdffc5372a29.jpg)
![cup 02](https://cloud.githubusercontent.com/assets/5188518/11748643/77f6a112-a029-11e5-9dd0-70d6ac548f2e.jpg)

Some samples, from the net. The medals come in 7 varieties in gold silver and bronze.
Is it something we are looking for?

**Comment by minj** [comment]
not sure..

perhaps the first one for `trophyCupDivisional*A` (with roman numerals), last 3 for `trophyTournamentWinner*`, and the other 3 for `trophyCupDivisionalB*`

in any case they have a different style and need to be adapted somehow

**Comment by ratsrats** [comment]
![cups 02](https://cloud.githubusercontent.com/assets/5188518/11836376/aa3b738c-a3da-11e5-9423-6e241c2fc6e1.png)
![cups 03](https://cloud.githubusercontent.com/assets/5188518/11836377/aa3fecdc-a3da-11e5-99e4-d1ba2be9d9cc.png)
![cups 04](https://cloud.githubusercontent.com/assets/5188518/11836378/aa481894-a3da-11e5-86bc-35523d3356c0.png)

adapted, looking for improvement.

**Comment by ratsrats** [comment]
![cups 02a](https://cloud.githubusercontent.com/assets/5188518/11836751/fe75031c-a3dc-11e5-8ac6-7f10a00dff72.png)

added contrast this one

**Comment by ratsrats** [comment]
![cups 02b](https://cloud.githubusercontent.com/assets/5188518/11836802/405355ae-a3dd-11e5-83c8-f3475126e49b.png)
![cups 03](https://cloud.githubusercontent.com/assets/5188518/11836803/405d0ebe-a3dd-11e5-9d79-2fba6103b75c.png)
![cups 04](https://cloud.githubusercontent.com/assets/5188518/11836804/40675950-a3dd-11e5-9ef3-41e4ed141c43.png)

to see these together

**Comment by ratsrats** [comment]
![cup 1c](https://cloud.githubusercontent.com/assets/5188518/11836867/af012d14-a3dd-11e5-8d66-f6a139495eb9.png)
![cup 1b](https://cloud.githubusercontent.com/assets/5188518/11836868/af0a777a-a3dd-11e5-9507-78cb930232b2.png)
![cup 1a](https://cloud.githubusercontent.com/assets/5188518/11836869/af129c16-a3dd-11e5-8a2f-0f15908aca45.png)

These were mend to be different. May be we should look further?

**Comment by ratsrats** [comment]
![medaille 01 gold b](https://cloud.githubusercontent.com/assets/5188518/11837494/948669aa-a3e1-11e5-9293-90c8d4b4f60d.png)
![medaille 01 gold a](https://cloud.githubusercontent.com/assets/5188518/11837495/948acd7e-a3e1-11e5-83f5-c11a5b854a01.png)

with roman numerals, the same medal in 2 versions.

**Comment by minj** [comment]
I like ![](https://cloud.githubusercontent.com/assets/5188518/11837495/948acd7e-a3e1-11e5-83f5-c11a5b854a01.png) on the whole but isn't it too different from the old ones? Perhaps something like that would suit `trophySupporterWeekTrophyWinner` instead.

You don't like my ideas in the TODO list?

**Comment by ratsrats** [comment]
I like almost all your ideas : ) 

That is not the point. You have to educate me . . . I wouldn't know how a trophySupporterWeekTrophyWinner would look like. 
Now I have looked again into this thread. The August 9th post . . .  ; )

I'll try and bring it all closer together.

**Comment by ratsrats** [comment]
Recap:

Well the TODO list pretty much covers it:

    trophyCupB* cups are of the same rank and need to look the same, only different colors (red, green, blue). I think they should all have one 'ear' like the red one above so blue & green should be remade.
![b1](https://cloud.githubusercontent.com/assets/5188518/11842967/35232b30-a407-11e5-9513-547431180baa.png)     trophyCupDivisionalB* cups should also be of the same (but lower rank) and I think they should have no ears but with one pom pom like the blue one above so green & red should be done accordingly.
 ![cupa](https://cloud.githubusercontent.com/assets/5188518/11843038/90db1906-a407-11e5-88ed-ae1337bf3817.png)   trophyCupDivisional*A should look like trophyCupDivisionalB* but purple and with roman numerals like HT originals up above.
![tr sup](https://cloud.githubusercontent.com/assets/5188518/11843076/d21e566c-a407-11e5-81f6-f3d59e4187ed.png)    need some new ideas for trophySupporterWeekTrophyWinner and trophyTournamentWinner*

Is this correct?

**Comment by ratsrats** [comment]
It takes a lot of time, before I can see what you mean. I had thought the already delivered cups were ready. But it seems you want to use those, with some tweaking.
and you need something new for trophySupporterWeekTrophyWinner and trophyTournamentWinner*

**Comment by ratsrats** [comment]
Here is my proposal:

![cup 1b](https://cloud.githubusercontent.com/assets/5188518/11843394/a987e9fa-a409-11e5-98e6-8953b246d383.png)  trophySupporterWeekTrophyWinner

![medaille 01 gold b](https://cloud.githubusercontent.com/assets/5188518/11843417/cc001ea8-a409-11e5-812d-0cb50d8f7eb4.png)  trophyTournamentWinner*

**Comment by ratsrats** [comment]
Here is my proposal

trophyCupDivisionalB* cups

![cupb2_10_03](https://cloud.githubusercontent.com/assets/5188518/11843570/9af49838-a40a-11e5-99c8-f3b108fc7691.png)
![cupb2_10_03blue](https://cloud.githubusercontent.com/assets/5188518/11843868/22903fa8-a40c-11e5-9db9-65bade7cd8e0.png)
![cupb2_10_03green](https://cloud.githubusercontent.com/assets/5188518/11843872/25901b42-a40c-11e5-9855-b36a23bb5972.png)

**Comment by ratsrats** [comment]
Here is my proposal

 trophyCupDivisionalA (I am afraid, all are mixed up)

![cupb3_10_03 green](https://cloud.githubusercontent.com/assets/5188518/11844301/59a68aae-a40e-11e5-8879-f025e13b094c.png)
![cupb3_10_03 red](https://cloud.githubusercontent.com/assets/5188518/11844306/5fbfef84-a40e-11e5-9b3d-26eacd216bd4.png)
![cupb3_10_03](https://cloud.githubusercontent.com/assets/5188518/11844307/65b46e38-a40e-11e5-88b1-ce2621ad3215.png)

**Comment by ratsrats** [comment]
Here is my proposal

![cupb2_10_03 purple](https://cloud.githubusercontent.com/assets/5188518/11844392/e29aadf4-a40e-11e5-9ade-cec8ec4fc6fa.png)

**Comment by ratsrats** [comment]
May be mixed up, but I hope you will be able to find what you have been looking for. There need to be some Roman numbers to. I'll see to that as well.

**Comment by ratsrats** [comment]
About the roman numbers.

For e.g. VIII A, there is simply not enough room to experiment with fonts. 

In Arial which is a good simple font for this, it takes 11 pix for VIII. The last "I" is semi covered by the top of the cup. May be some coloring experimenting could improve the feeling.

![trophycupnational_8a - a](https://cloud.githubusercontent.com/assets/5188518/11844728/9394e524-a410-11e5-8ef7-baa9998059ea.png)

The Roman VIII is now in lighter green then before.

**Comment by ratsrats** [comment]
Please include pictures to make clear what you are talking about. I am not so good in the connection of names. Programming, you can do best, would drive me crazy, not to see what it is about.

**Comment by minj** [comment]
Wow, so much to look at it. Thanks.

---
![cup 1b](https://cloud.githubusercontent.com/assets/5188518/11843394/a987e9fa-a409-11e5-98e6-8953b246d383.png)
Any chance to make this one purple for `trophySupporterWeekTrophyWinner`?

---
![medaille 01 gold b](https://cloud.githubusercontent.com/assets/5188518/11843417/cc001ea8-a409-11e5-812d-0cb50d8f7eb4.png)
When I write `trophyTournamentWinner*` the star symbol means there are more of them and they should have a similar style. Comment on August 9th shows there are 3 trophies of this type. Do you propose to use the same for all of them?

I liked these on principal:
![cups 02b](https://cloud.githubusercontent.com/assets/5188518/11836802/405355ae-a3dd-11e5-83c8-f3475126e49b.png) ![cups 03](https://cloud.githubusercontent.com/assets/5188518/11836803/405d0ebe-a3dd-11e5-9d79-2fba6103b75c.png) ![cups 04](https://cloud.githubusercontent.com/assets/5188518/11836804/40675950-a3dd-11e5-9ef3-41e4ed141c43.png)
 but they don't really fit the general 'cartoonish' style. Anything you could here?

---
![cupb2_10_03](https://cloud.githubusercontent.com/assets/5188518/11843570/9af49838-a40a-11e5-99c8-f3b108fc7691.png) ![cupb2_10_03blue](https://cloud.githubusercontent.com/assets/5188518/11843868/22903fa8-a40c-11e5-9db9-65bade7cd8e0.png) ![cupb2_10_03green](https://cloud.githubusercontent.com/assets/5188518/11843872/25901b42-a40c-11e5-9855-b36a23bb5972.png)
These match the description of `trophyCupB*` exactly so consider them done.

---
![cupb2_10_03](https://cloud.githubusercontent.com/assets/5188518/11844301/59a68aae-a40e-11e5-8879-f025e13b094c.png) ![cupb3_10_03 red](https://cloud.githubusercontent.com/assets/5188518/11844307/65b46e38-a40e-11e5-88b1-ce2621ad3215.png)

These match the description of `trophyCupDivisionalB*` exactly but I need a red one as well. This seems too purple/pink (compare to the previous section):
![cupb3_10_03](https://cloud.githubusercontent.com/assets/5188518/11844306/5fbfef84-a40e-11e5-9b3d-26eacd216bd4.png)

---
Now, considering `trophyCupDivisional*A`....
Your proposal: ![](https://cloud.githubusercontent.com/assets/5188518/11844728/9394e524-a410-11e5-8ef7-baa9998059ea.png)
I don't think there is any need to write 'A' there, so `VIII A` should just be `VIII`.

![](https://cloud.githubusercontent.com/assets/5188518/11837495/948acd7e-a3e1-11e5-83f5-c11a5b854a01.png) could be used here instead. Just the number is wrong. We need `VII`, `VIII` and `IX`

But in any case I think you are right, roman numerals are too wide. How about another proposal:

1. Let's take this: ![](https://cloud.githubusercontent.com/assets/5188518/11844392/e29aadf4-a40e-11e5-9ade-cec8ec4fc6fa.png)
1. remove the 'ear'
1. add regular numbers: `7`, `8`, `9` in the center of image (on top of the cup)

The numbers should look 'engraved'. Compare to: 
![](https://camo.githubusercontent.com/59fcd587aa9059cdc827157fbc4e1553b7177877/68747470733a2f2f73746167652e686174747269636b2e6f72672f4170705f5468656d65732f5374616e646172642f696d616765732f74726f70686965732f637570446976392e706e67)

We are half-way there. Keep them coming :+1

**Comment by ratsrats** [comment]
From top to bottom:

![cup 1b purple](https://cloud.githubusercontent.com/assets/5188518/11873631/e13bf0c6-a4db-11e5-9d0c-f8c1b5f60859.gif)
skip the next
![cupb3_10_03 red](https://cloud.githubusercontent.com/assets/5188518/11873667/12733fb4-a4dc-11e5-8e9c-2e22af8be018.png)
![trophycupnational_7a](https://cloud.githubusercontent.com/assets/5188518/11873836/d8fca436-a4dc-11e5-85b6-7d50cbb67ea6.png)
![trophycupnational_8a - a](https://cloud.githubusercontent.com/assets/5188518/11873843/dc048e82-a4dc-11e5-8fa3-b5baf2605e8f.png)

**Comment by ratsrats** [comment]
In the 1,2,3 approach I cannot understand why you like to have the ear removed. 

![cupb2_10_03 purple 7](https://cloud.githubusercontent.com/assets/5188518/11874558/b20de8cc-a4e0-11e5-9e97-4cf422eaf3f9.png)
![cupb2_10_03 purple 8](https://cloud.githubusercontent.com/assets/5188518/11874559/b3feb3dc-a4e0-11e5-8be7-fb16485ef629.png)
![cupb2_10_03 purple 9](https://cloud.githubusercontent.com/assets/5188518/11874564/b623cbfc-a4e0-11e5-8dde-77e7197715e0.png)

The fiddling with the number is unpleasant. There is very little to work with. Nice would be a large number and the cup partly covering this number. On 20x30 pix my possibilities are almost exhausted. To give these number the same impact, I had to change color ; )

**Comment by ratsrats** [comment]
To jump on one branch: 

![medaille 01 gold b vii](https://cloud.githubusercontent.com/assets/5188518/11874804/c9bb27c2-a4e1-11e5-828a-ded40924a5e9.png)
![medaille 01 gold b viii](https://cloud.githubusercontent.com/assets/5188518/11874809/cca8ee56-a4e1-11e5-8502-cd49cc07b9b6.png)
![medaille 01 gold b ix](https://cloud.githubusercontent.com/assets/5188518/11874810/cf1733c8-a4e1-11e5-99bf-ec0d963c16c6.png)

**Comment by ratsrats** [comment]
As you can see, the new HT Cups are larger than these old ones.

I can understand why that move has been made. May be you too. If the burden of all the Foxtrick programming is becoming heavy, I can understand that too. Foxtrick is an amazingly clever add on. Do not waste too much time over these cup details.

A new approach would end this fiddling ; ) too. My suggestion would be:
Let's get it over with, any way, quick and dirty has been the way all along : ) Until a real solution is presented. It could be in the form of enlarging the canvas for these cups and have someone do the designing on the cups or find a funny way to present all these cups.

I do not think the new HT Cups are a success, certainly not in comparison with this lovely old cup we are raping all the time.

**Comment by minj** [comment]
I don't mind having them larger. But do you want to make larger versions for all the old ones then? :P 

The reason I want the ear removed is that `trophyCupDivisional*A` cups should not look better than `trophyCupB*`. Lower divisions are boring. `trophyCupB*` on the other hand includes teams from all over the country, so what if they lost once. I'm trying to win one at this very moment :laughing:  I actually like the ones with indo-arabic numerals that you 'fiddled with'. Just the ear needs to go imho.

I'm marking `trophyCupB*` as done. `trophySupporterWeekTrophyWinner` is OK as well. No better ideas for `trophyCupDivisional*A` or `trophyTournamentWinner*`?

**Comment by ratsrats** [comment]
There has been no email with your post. I only just have seen it

**Comment by ratsrats** [comment]
" I'm trying to win one at this very moment " if you win, you may decide the looks ; )

**Comment by ratsrats** [comment]
I'll have to see what can be done. 
"I don't mind having them larger." But do you want to make larger versions for all the old ones then? :P Upscaling is always possible. Tell me what canvas size looks good to you. May be the current HT Cups canvas?
Give me some time here. If you become impatient, just let me know.

**Comment by minj** [comment]
> Tell me what canvas size looks good to you. May be the current HT Cups canvas?
Sure

**Comment by ratsrats** [comment]
Happy new year!

![cups 02 drawing 10 _ 01](https://cloud.githubusercontent.com/assets/5188518/12142561/5c5016fa-b479-11e5-9d78-ca89d904840a.png)

This is may not be the variation you are looking for.

**Comment by minj** [comment]
what is that supposed to be?

**Comment by ratsrats** [comment]
It's a cup, at your perusal.

Trying to fake a cartoon style . . . (The size is not right.) 

I do not like the new HT cups either. We have only one excellent old one to work with. So something new has to happen for all those new cups. We cannot continue 'building' on this one old sample alone.

**Comment by minj** [comment]
Yes, I can see it's a cup. The question is, which type of cup you propose it to be?

I like it I guess but those tiny details will not look OK when it's resized.

**Comment by ratsrats** [comment]
![cups 02 drawing 10 _ 01 34px](https://cloud.githubusercontent.com/assets/5188518/12148859/06f4d9d4-b4a1-11e5-8587-d908b8a93658.png)

this how it looks like in 34 px wide.

**Comment by minj** [comment]
well since I got [randomed](https://www.hattrick.org/goto.ashx?path=/Club/Matches/Match.aspx?matchID=555206924) out of the wretched B cup. I don't care about this any more :frowning:

**Comment by ratsrats** [comment]
Why didn't you more attack over the wings, that seemed to be a better option . . .
Some opponent, impressive defence and longshot capability!
Most impressive attacking rates, before the send off! (and after still)

Shit happens.

Very nice team you have, well build.

**Comment by minj** [comment]
> Why didn't you more attack over the wings, that seemed to be a better option

What?! He has divine+++ defence on all sectors (distributed equally, I should add), why should I favor wings? Look at my chances, all of them are on the center. You can't do shit without center in higher divisions. If anything, I should have had more of it since I was way too weak there after the red card.

I won the middle, had ~divine attacks, seems that's all you need to kick a stupid LS team\* in the nuts. 

**NAAH!!!**

You need random to have your back.

\* Disclosure: I absolutely detest all LS teams to infinity and beyond

**Comment by minj** [comment]
As for my team, it seems I must buy overtrained IMs or I am doomed. This all would have been decided with an overwhelming possession. I have a love/hate relationship with Hattrick and these days the latter tends to take over.

That seems strange since I am now more successful than ever. I just don't know what else to do with my team to improve. I'm training SP for more than a season without any youth trainees lol. The finances are low-ish but improving a bit from regular income. Yet my players are aging and seem to be pushed to the limit (I trained some of them in 4 skills) but I don't want to sell them since the market is crap for old players, yet there is always a bidwar when you need something.

This is all a vicious cycle of frustration... but I am getting ahead of myself and off topic...

**Comment by minj** [comment]
I guess since I've finally won one, I should fix this issue soon :D

**Comment by ratsrats** [comment]
No trouble too much, I'll do anything to produce the best cup ever for your collection : )

Just give me a hint what you are looking for. May be the old Jules Rimet World Cup . . . that was stolen in 1983.
![jules rimet 02](https://cloud.githubusercontent.com/assets/5188518/15706755/6c8d0790-27f5-11e6-8ccd-f2bca3a9442d.jpg)
![jules rimet a 01](https://cloud.githubusercontent.com/assets/5188518/15706754/6c8741d4-27f5-11e6-8cca-08deecd4ab2c.jpg)
